x/cash/dynamicfee.go

```diff
@@ -0,0 +1,235 @@
+package cash
+
+import (
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/gconf"
+	"github.com/iov-one/weave/x"
+)
+
+/*
```

Comment:
Awesome!  :dancer: 
This comment section and the one in `x/cash/staticfee.go` are extremly helpful! 

How about moving it to the very top to become part of this [package documentation](https://github.com/golang/go/wiki/CodeReviewComments#package-comments)? :nail_care:

Reply:
Sure. Go for it!